### PR TITLE
feat: [258] Added app theme text styles to the text-style parser

### LINKED ISSUE #258: feat: Ability to use app theme `TextStyles`
**Description**

Right now the TextStyles are hardcoded in the text property. There should be a way to use theme text style in the text property.

```
"textStyle": "displayLarge"
```

### PR DESCRIPTION
**Pull request by i-asimkhan**
## Description
Added app theme text styles to the text-style parser

## Related Issues

Closes #258 

## Type of Change

- [X] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Code refactor
- [ ] Build configuration change
- [ ] Documentation
- [ ] Chore

### CODE REVIEW

**Comment by divyanshub024** [comment]
@i-asimkhan Can you run `flutter pub get` in mirai_gallery?

File: examples/mirai_gallery/macos/Podfile.lock [hunk 1]
```diff
@@ -13,4 +13,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 236401fc2c932af29a9fcf0e97baeeb2d750d367
 
-COCOAPODS: 1.14.3
+COCOAPODS: 1.12.1
```
Comment (divyanshub024) [inline examples/mirai_gallery/macos/Podfile.lock:16]: We should not downgrade the pods. Can you update your COCOAPODS?

File: packages/mirai/lib/src/action_parsers/mirai_dialog_action/mirai_dialog_action.g.dart [hunk 2]
```diff
@@ -40,5 +40,4 @@ Map<String, dynamic> _$$MiraiDialogActionImplToJson(
 const _$TraversalEdgeBehaviorEnumMap = {
   TraversalEdgeBehavior.closedLoop: 'closedLoop',
   TraversalEdgeBehavior.leaveFlutterView: 'leaveFlutterView',
-  TraversalEdgeBehavior.parentScope: 'parentScope',
```
Comment (divyanshub024) [inline packages/mirai/lib/src/action_parsers/mirai_dialog_action/mirai_dialog_action.g.dart:43]: We should not remove it. Can you run `melos build ` here?

**Review by divyanshub024 — CHANGES_REQUESTED** [review]

File: packages/mirai/lib/src/action_parsers/mirai_dialog_action/mirai_dialog_action.g.dart (hunk 2 above)
Comment (i-asimkhan) [reply packages/mirai/lib/src/action_parsers/mirai_dialog_action/mirai_dialog_action.g.dart:43]: Hey I am updating my local cocoa pods that will fix these,

Comment (i-asimkhan) [reply packages/mirai/lib/src/action_parsers/mirai_dialog_action/mirai_dialog_action.g.dart]: Fixed.

**Review by divyanshub024 — APPROVED** [review]
LGTM!

**Pull request merged**